src/lib/dom.iterable.d.ts

```diff
@@ -1,30 +1,30 @@
 /// <reference lib="dom" />
 
 interface DOMTokenList {
-    [Symbol.iterator](): IterableIterator<string>;
+    [Symbol.iterator](): IterableIterator<string, void>;

```

Comment:
Presumably this change is going to be reflected in [the dom lib generator](https://github.com/microsoft/TypeScript-DOM-lib-generator), too?

Reply:
This file isn't one of the generated ones, actually, so I don't think the generator has to change unless one of the `*.generated.*.d.ts` files are edited.

Reply:
Yes, I plan to do that as well.

Reply:
The generator will need to change so that we have the correct types on any DOM collections produced by the generator.

Reply:
That ends up being a bit tricky if we're also shipping DOM types in a standalone package since older TS wouldn't support `TReturn` and `TNext`.

Reply:
> Presumably this change is going to be reflected in [the dom lib generator](https://github.com/microsoft/TypeScript-DOM-lib-generator), too?

https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1713